<launch>
    <node name="galaxy_camera" pkg="galaxy_camera" type="galaxy_camera" output="screen">
        <param name="image_width" value="1280"/>
        <param name="image_height" value="1024"/>
        <param name="pixel_format" value="bgr8"/>
        <param name="camera_info_url" value=""/>
        <param name="camera_frame_id" value="pitch_camera"/>
    </node>
    <node ns="galaxy_camera" name="image_proc" pkg="image_proc" type="image_proc" respawn="true"/>

</launch>
